a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -489,16 +482,14 @@ <h4>Examples</h4>
 					<aside class="example" title="Descriptive explanations">
 						<ul>
 							<li data-localization-id="nonvisual-reading-readable" data-localization-mode="descriptive"
-								>All content is available in generated read aloud speech and electronic braille</li>
+								>All content is available in generated read aloud speech and generated refreshable  braille</li>
```

Comment:
This still sounds like it's implying that the speech and braille have been pre-generated and are available to everyone. These options only exist if the device has those capabilities, and you may only be able to get one or the other. It would make more sense to me to say "All content can be generated as read aloud speech or dynamic braille"

Similarly, for the next two I'd go with

- Portions of the content may not be generable as read aloud speech or dynamic braille
- Not all of the content can be generated as read aloud speech or dynamic braille